specification/vmware/resource-manager/Microsoft.AVS/preview/2021-01-01-preview/vmware.json

```diff
@@ -4337,6 +4553,99 @@
         }
       }
     },
+    "DatastoreList": {
+      "description": "A paged list of datastores",
+      "properties": {
+        "value": {
+          "description": "The items on a page",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/Datastore"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page if any",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Datastore": {
+      "description": "A datastore resource",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "The properties of a datastore resource",
+          "$ref": "#/definitions/DatastoreProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "DatastoreProperties": {
+      "description": "The properties of a datastore",
+      "properties": {
+        "provisioningState": {
+          "description": "The state of the datastore provisioning",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Cancelled",
+            "Deleting",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "DatastoreProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "netAppVolume": {
+          "description": "Resource ID of the Azure NetApp Files volume",
+          "$ref": "#/definitions/NetAppVolumeProperties"
+        },
+        "diskPoolVolume": {
+          "description": "Resource ID of the iSCSI target from a Storage Pool and LUN to be used",
+          "$ref": "#/definitions/DiskPoolVolumeProperties"
+        }
+      }
+    },
+    "NetAppVolumeProperties": {
+      "description": "A model which holds properties of an Azure NetApp Files volume of Microsoft.NetApp provider",
+      "properties": {
+        "nfsProviderIp": {
+          "type": "string",
+          "description": "IP address of the NFS provider"
+        },
+        "nfsFilePath": {
+          "type": "string",
+          "description": "File path through which the NFS volume is exposed by the provider"
+        }
+      },
+      "x-ms-azure-resource": true
```

Comment:
This is not a resource . http://azure.github.io/autorest/extensions/#x-ms-azure-resource